packages/litexa/src/parser/litexa.pegjs

```diff
@@ -797,13 +799,23 @@ SayStatement
   }
 
 /* litexa [reprompt]
-Specifies the reprompt that will be installed during
-the next response. The content is specified in the
-[Say String](#say-string) format.
+Adds a [Say String](#say-string) to the pending skill response's reprompt speech.
+Same as the [say](#say) statement, any reprompted fragments are accumulated.
+```coffeescript
+reprompt "Hello"
+reprompt "World"
+# would result in Alexa reprompting "Hello World"
+```
 
-Note: Unlike [say](#say) statements, reprompt statements
-are *not* accumulative; only the most recent reprompt statement
-will be included in the response.
+Alternatively, writing a standalone `reprompt` will add the entire pending
+response's `say` speech to the reprompt.
+
+```coffeescript
+say "one"
+say "two"
+reprompt # would result in Alexa reprompting "one two three"
+say "three"
+```
```

Comment:
Same as per above comment, this "echo" reprompt was ultimately removed, so wouldn't include these changes in the PR, to avoid confusion.